src/main/java/uk/gov/hmcts/reform/iahearingsapi/infrastructure/hmc/UnNotifiedHearingsProcessor.java

```diff
@@ -48,7 +48,7 @@ public void run() {
     public void processUnNotifiedHearings() {
 
         UnNotifiedHearingsResponse unNotifiedHearings = hearingService.getUnNotifiedHearings(
-            LocalDateTime.now().minusDays(3));
+            LocalDateTime.now().minusMonths(10));
```

Comment:
I pushed this accidentally in my PR, already removed. Is this intended or rebase issue?